### PR TITLE
Additon: Add /gd as supplement to /warp garden

### PR DESCRIPTION
**Pull request by RayDeeUx**

### CODE REVIEW

**Comment by Absterge** [comment]
Not against this being added per se, just that it's already possible in both Skytils and DSM. Is this really needed?

**Comment by hannibal002** [comment]
I would either implement the full logic of https://discord.com/channels/997079228510117908/1149744503234764821 directly, or nothing at all. Currently, this is just Skytils with fewer options.

**Comment by hannibal002** [comment]
As cal found out, it is the feature "optimised font renderer" from patcher

File: src/main/java/at/hannibal2/skyhanni/features/commands/WarpGdCommand.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class WarpGdCommand {
+
+    @SubscribeEvent
+    fun onSendPacket(event: PacketEvent.SendEvent) {
+        if (!LorenzUtils.inSkyBlock) return
+        if (!SkyHanniMod.feature.commands.supplementWarpGarden) return
+
+        val packet = event.packet
+        if (packet is C01PacketChatMessage && packet.message.lowercase() == "/gd") {
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/commands/WarpGdCommand.kt:17]: We should start using a "player send message to server" event

**Comment by hannibal002** [comment]
If this pr prepared for additional garden warp features (e.g. with key bind that only works inside the garden or hub) then I would merge it.

But just as a command alias, this is useless compared to Skytils' feature.

**Pull request closed without merging**